pkgs/development/python-modules/badldap/default.nix

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
`badlap` change should not be in the `python3Packages.asyauth-bad -> python3Packages.badauth` commit.